migrations/004.do.inventory-update.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TYPE enum_material AS ENUM('Dragon Talon', 'Dragon Hide', 'Dragon Fang');
+CREATE TABLE materials (
+    type enum_material NOT NULL,
+    quantity INT,
+    PRIMARY KEY(player_id, material_id)
+);
+CREATE TYPE enum_geartype AS ENUM('DragonsBite');
+CREATE TABLE equipment (
+    id SERIAL PRIMARY KEY,
+    player_id INT NOT NULL references players(id) ON DELETE CASCADE,
+    type enum_geartype NOT NULL
+);
+CREATE TABLE equipment_mods (
+    id SERIAL PRIMARY KEY,
+    equipment_id INT NOT NULL REFERENCES equipment (id) ON DELETE CASCADE,
```

Comment:
Nothing is gained from having a second table here, it's a 1 to 1 relationship: every equipment has and must have a list of mods. 

```suggestion
    type enum_geartype NOT NULL,
```